torch/csrc/jit/serialization/export.cpp

```diff
@@ -870,6 +870,44 @@ std::tuple<std::string, RawDataExportMap> export_onnx(
       graph_encoder.get_raw_data_export_map());
 }
 
+std::tuple<::ONNX_NAMESPACE::ModelProto, RawDataExportMap> export_onnx_opt(
```

Comment:
what does `opt` in the function name stand for?
Can we update `export_onnx` to call this new function, such to reduce duplicate code?

Reply:
Are the new changes enough to reduce the duplicate code, or is there another possible method to do this